# this msg publishes the bounding boxes comes from inference script
Header header
int32 no_of_boxes
string[] label_and_confidence
box_coordinates[] coordinates
# box coordinates are of the form (x, y, w, h)

